prow/plugins/releasenote/releasenote_test.go

```diff
@@ -113,10 +115,10 @@ func TestReleaseNoteComment(t *testing.T) {
 				Assignees:   []github.User{{Login: "r"}},
 			},
 		}
-		if tc.isAuthor {
+		if tc.isMember {
+			ice.Comment.User.Login = "member"
+		} else if tc.isAuthor {
 			ice.Comment.User.Login = "a"
-		} else if tc.isReviewer {
```

Comment:
It would be same as [original](https://github.com/kubernetes/test-infra/blob/master/prow/plugins/releasenote/releasenote_test.go#L116-L120) but on [this](https://github.com/kubernetes/test-infra/blob/master/prow/plugins/releasenote/releasenote_test.go#L118) line, change `tc.isReviewer` to `tc.isMember`.